src/api.js

```diff
@@ -660,7 +661,8 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
     * by a semicolon)
     *
     * ## Example use
-    * We have the following table, named *test* :
+    * We will create following table, named *test* and query it with a
```

Comment:
```suggestion
    * We will create the following table, named *test* and query it with a
```

Reply:
applied suggestion in commit 6e5024ed6c522e2719bb37e9418f708e29cf3786